CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 
 * Remove `display: none` rules from component print stylesheets, and use the `govuk-!-display-none-print` class instead. ([PR #1561](https://github.com/alphagov/govuk_publishing_components/pull/1561))
 * Update link styles for show password component ([PR #2074](https://github.com/alphagov/govuk_publishing_components/pull/2074))
+* Update share links to have new underline styles ([PR #2073](https://github.com/alphagov/govuk_publishing_components/pull/2073))
```

Comment:
I think this branch needs a rebase, not sure you meant to add this line to the `CHANGELOG`.

Reply:
I should've mentioned that in the PR description. https://github.com/alphagov/govuk_publishing_components/pull/2073 didn't include a changelog line and I merged it on Chris' behalf before spotting it. I'll make a note in the PR desc. (This branch also probably needs a rebase anyway).

Reply:
Ahhh right I see. That makes sense. I thought you'd added it by accident.

Reply:
I think it's a reasonable assumption that it was a mistake. Cheers for noting it 👍